src/image-tags.js

```diff
@@ -11,7 +11,14 @@ export function ImageTags(seoConfig, config, pageType, data, {url}) {
     return [];
 
   const tags = [];
-  const image = new FocusedImage(story["hero-image-s3-key"], story["hero-image-metadata"] || {});
+  let image = new FocusedImage(story["hero-image-s3-key"], story["hero-image-metadata"] || {});
+
+  if(url.query && url.query.cardId){
+      const { metadata = {} } = story.cards.find(card => card.id === url.query.cardId);
+      if(metadata  && !isEmpty(metadata)){
+          image = new FocusedImage(metadata['social-share'].image.key, metadata['social-share'].image.metadata || {});
```

Comment:
if `metadata['social-share']` is undefined?  or `metadata['social-share'].image` ?